### DISCUSSION TITLE
OpenCl on jetson xavier nx

### DISCUSSION THREAD

**Discussion by yrodriguez31** [OE4T/meta-tegra#1605] [Q&A]
Hello 

I searched in the discussions to find a way to include OpenCL in a Yocto build, but I couldn't find any information about that. Generally, the Yocto recipes I found are for OpenCL on i.MX but nothing for other targets.

Is there a way to include OpenCL with meta-tegra and compile OpenCV with the correct flag to indicate that we have OpenCL?

Thanks.

**Comment by kekiefer** [comment]
[OpenCL isn't supported](https://forums.developer.nvidia.com/t/opencl-support-on-jetson-agx-orin/257122/3) by NVIDIA's L4T releases. CUDA is the GPU compute language of choice for these platforms, and that can also work with OpenCV.

**Comment by r8d8** [comment]
https://forums.developer.nvidia.com/t/can-the-xavier-run-opencl-applications/70262/15
